admin/commands/state_synchronization/execute_script.go

```diff
@@ -3,31 +3,75 @@ package state_synchronization
 import (
```

Comment:
let's just remove this command. this is from when we were first implementing script execution. now that the APIs fully support local scripts, don't need it anymore